parachain/runtime/runtime-tests/src/lib.rs

```diff
@@ -1,3 +1,13 @@
 #![cfg(test)]
+#![allow(dead_code)]
+#![feature(exclusive_range_pattern)]
 
+mod bitcoin_data;
+mod parachain;
 mod relaychain;
+
+#[cfg(any(feature = "with-interlay-runtime", feature = "with-kintsugi-runtime",))]
+mod setup;
+
+#[cfg(any(feature = "with-interlay-runtime", feature = "with-kintsugi-runtime",))]
```

Comment:
Can we have one of the features on by default? I.e. by default use kintsugi, and if `with-interlay-runtime` is enabled, use that instead? That way a normal `cargo test` will include runtime tests